eahub/templates/eahub/profile.html

```diff
@@ -22,6 +22,15 @@
     </div>
     <div class="col-xs-12 col-md-6">
       <h1 class="word-break">{{ profile.name }}</h1>
+      {% if profile.linkedin_url %}
+      <a href="{{ profile.linkedin_url }}"><i class="fab fa-linkedin-in"></i></a>
```

Comment:
This doesn't act quite as I would expect. If I enter "www.linkedin.com" it creates the link eahub.org/seb/profile/www.linkedin.com. Only if you add "https://" does it create the expected URL

Reply:
If I use `URLField` and enter `www.linkedin.com` or `linkedin.com`, Django adds `http://` to the beginning, so this isn't a problem.